src/components/elements/Inputs/Input/Input.tsx

```diff
@@ -46,31 +55,66 @@ const Input = forwardRef(
       className,
       iconButtonProps,
       hideErrorMessage,
+      customVariant,
+      labelClassName,
+      descriptionClassName,
+      labelVariant,
+      readOnly,
+      format,
+      sufixLabelView,
+      classNameContainerInput,
+      classNameError,
       ...inputWrapperProps
     }: InputProps,
     ref?: Ref<HTMLInputElement>
   ) => {
-    const { label, description, containerClassName, error, required, feedbackRequired, ...inputProps } =
-      inputWrapperProps;
+    const {
+      label,
+      description,
+      descriptionFooter,
+      containerClassName,
+      error,
+      required,
+      feedbackRequired,
+      ...inputProps
+    } = inputWrapperProps;
     const id = useId();
-
+    const customVariantClasses = customVariant ? customVariant : {};
```

Comment:
`const customVariantClasses = customVariant ?? {}`
or simply
`customVariant ??= {}` (and use `customVariant` instead of `customVariantClasses` below)

Or now that I think more about it, the declaration in the props above could just be `customVariant = {}`

Reply:
done